stats.py

```diff
@@ -504,7 +523,7 @@ def execute_builder(b):
 
 
 # need docs
-def build_result(df, ignore_errors=False, blame=False):
+def build_result(df, ignore_errors=False, blame=False, only=[]):
```

Comment:
Using mutable structures (like lists) as default parameters is evil, consider to review that. You can use tuples.

Reply:
Thats true, maybe just none and assert its iterable after calling ?

Reply:
Yes, you can use None too. But probably tuple is better since you expect to receive a iterable object.

Reply:
Alright !